frontend/src/views/Transfers/ViewTransfer/ViewTransfer.jsx

```diff
@@ -224,7 +236,31 @@ export const ViewTransfer = () => {
     ],
     Recommended: [
       {
-        ...saveDraftButton(t('saveBtn')),
+        ...redOutlinedButton(t('transfer:refuseTransferBtn'))
+      },
+      {
+        ...outlinedButton(t('transfer:returnToAnalystBtn')),
+        handler: () =>
+          setModalData({
+            primaryButtonAction: () =>
+              updateTransfer({
+                newStatus: 4,
```

Comment:
Can you think of a way so we can use status names rather than ids here?

Reply:
would the goal be to make it more readable or to pass the actual string value?

few options come to mind:

1.
```
export const statuses = {
  ...
  sent: {
    id: 4,
    label: 'Sent'
  }
  ...
}
```
and then pass in `statuses.sent.id`.

2. create a util function that maps a string value to a status id
3. using the actual string value of a status and handling the mapping on the backend

Reply:
I think we should use the string value, and then map that to the Enum value on the backend. Updating a transfer on the backend has the same issue, checking against the ids without knowing their value. I will create a new ticket for this refactor. Please just fix the existing conflicts and we'll close this pr.